tests/integration/README.md

```diff
@@ -0,0 +1,23 @@
+# Integration tests for the Cachito
+
+This directory stores the integration tests for Cachito.
+
+## Configuration
+
+Input data for tests should be configured in a file `test_env_vars.yaml`. The file should be placed in top-level directory of this repository.
+
+See `tests/integration/example_test_env_vars.yaml` for a complete list of configuration options and examples.
+
+## Running tests
+
+Tests can be triggered from the top-level directory of this repository with:
+
+    tox -e integration
+
+The integration environment is not part of the default `tox` envlist.
+
+`REQUESTS_CA_BUNDLE` is passed in `tox.ini` for the `integration`
+environment in order to enable running the tests against Cachito instances which
+have self-signed certificates. Example usage:
```

Comment:
Not necessarily self-signed certs, but issue by a custom root certificate authority.

Reply:
changed